src/modules/tests/postComments.test.js

```diff
@@ -0,0 +1,17 @@
+import { postComments } from '../comments.js';
+
+test('should notify a comment has been posted', async () => {
+  const data = {
+    item_id: '52819',
+    username: 'Junaid',
+    comment: 'Hello',
+  };
+
+  const response = await postComments(
+    data.item_id,
+    data.username,
+    data.comment,
+  );
+
+  expect(response).toBe('Created');
```

Comment:
- [ ] Please you have one failing test so make sure that you fix it to ensure that the app is working perfectly :pray: 
  
  ![image](https://github.com/iamsjunaid/js-capstone-2/assets/92303891/36b75bf3-ff8c-4c76-b7f4-e65ce3f5546d)